packages/xo-web/src/xo-app/menu/index.js

```diff
@@ -281,6 +281,11 @@ export default class Menu extends Component {
             label: 'notificationsPage',
             extra: <NotificationTag />,
           },
+          isAdmin && {
+            to: 'xoa/support',
+            icon: 'menu-support',
+            label: 'supportPage',
+          },
```

Comment:
Should we hide it for the community version?

Reply:
What are we doing for the other pages?

Reply:
For example, we disabled all the inputs of the Updater page and added a message. We could do that here as well...

Reply:
Let's keep things as simple as possible, if we want to release today.

Reply:
Right, we can change this later.